TILApp/Views/CalendarView/CalendarViewController.swift

```diff
@@ -177,20 +84,16 @@ class CalendarViewController: UIViewController {
         $0.calendarWeekdayView.weekdayLabels[4].font = .boldSystemFont(ofSize: 20)
         $0.calendarWeekdayView.weekdayLabels[0].textColor = .darkGray
         $0.calendarWeekdayView.weekdayLabels[0].font = .boldSystemFont(ofSize: 20)
-        // 일요일 라벨의 textColor를 red로 설정
-        $0.calendarWeekdayView.weekdayLabels[6].textColor = .darkGray
-        $0.calendarWeekdayView.weekdayLabels[6].font = .boldSystemFont(ofSize: 20)
-
-        // 토요일 라벨 파란색으로
+        // 토요일 라벨
         $0.calendarWeekdayView.weekdayLabels[5].textColor = .darkGray
         $0.calendarWeekdayView.weekdayLabels[5].font = .boldSystemFont(ofSize: 20)
+        // 일요일 라벨
+        $0.calendarWeekdayView.weekdayLabels[6].textColor = .darkGray
+        $0.calendarWeekdayView.weekdayLabels[6].font = .boldSystemFont(ofSize: 20)
     }
 
     override func viewWillAppear(_ animated: Bool) {
-
-        navigationController?.navigationBar.isHidden = true
-
-
+        navigationController?.navigationBar.isHidden = false
     }
 
     override func viewDidLayoutSubviews() {
```

Comment:
```suggestion
    override func viewDidLayoutSubviews() {
        super.viewDidLayoutSubviews()
        calendarFlexView.flex.direction(.column).define {
            $0.addItem(calendarResultContent).alignItems(.center)
            $0.addItem(calendarView).marginLeft(20).marginRight(20).aspectRatio(1)
            $0.addItem().grow(1).define { flex in
                flex.view?.addSubview(tableView)
            }
        }

        calendarFlexView.pin.all(view.pin.safeArea)
        calendarFlexView.flex.layout()
        tableView.pin.all()
    }
```

tableView 말고 빈 UIView를 추가하였습니다. tableView 위치는 아래서 잡아줬어요.
이러면 바텀탭바 안 가릴거에요.